frontend/server/src/Validators.php

```diff
@@ -323,6 +323,49 @@ public static function validateNumberInRange(
         }
     }
 
+    /**
+     *
+     * @param mixed     $parameter
+     * @param string    $parameterName
+     * @param int|float|null $lowerBound
+     * @param int|float|null $upperBound
+     * @param boolean   $required
+     * @throws \OmegaUp\Exceptions\InvalidParameterException
+     */
+    public static function validateTimestampInRange(
+        $parameter,
+        string $parameterName,
+        $lowerBound,
+        $upperBound,
+        bool $required = true
+    ): void {
+        if (!self::isPresent($parameter, $parameterName, $required)) {
+            return;
+        }
+        if (!is_numeric($parameter)) {
+            throw new \OmegaUp\Exceptions\InvalidParameterException(
+                'parameterNotADate',
+                $parameterName
+            );
+        }
+        // Coerce $parameter into a numeric value.
+        $parameter = $parameter + 0;
```

Comment:
al contrario de `validateNumber` (donde el número puede ser un entero o un flotante), aquí siempre quieres que sea un entero:

```suggestion
        $parameter = intval($parameter )
```

y como ya es obvio lo que hace, puedes quitar L351